traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -548,6 +591,20 @@ var TableServersController = function(servers, $scope, $state, $uibModal, $windo
 		}
 	};
 
+	$scope.onQuickSearchChanged = function() {
+		$scope.gridOptions.api.setQuickFilter($scope.quickSearch);
+		localStorage.setItem("servers_quick_search", $scope.quickSearch);
+	};
+
+	$scope.onPageSizeChanged = function() {
+		const value = Number($scope.pageSize);
+		$scope.gridOptions.api.paginationSetPageSize(value);
+		localStorage.setItem("servers_page_size", value);
+	};
+
+	$scope.clearColFilters = function() {
+		$scope.gridOptions.api.setFilterModel(null);
```

Comment:
don't you also wanna clear the filters stored in `localStorage`?

Reply:
this dispatches an event that is already handled with:

```
		$scope.gridOptions.api.addEventListener("filterChanged", function() {
			localStorage.setItem("servers_table_filters", JSON.stringify($scope.gridOptions.api.getFilterModel()));
		});

```

Reply:
oh, alright.